lib/iris/tests/unit/fileformats/netcdf/test__load_aux_factory.py

```diff
@@ -53,8 +53,44 @@ def test_formula_terms_ap(self):
         self.assertEqual(factory.surface_air_pressure, self.ps)
 
     def test_formula_terms_a_p0(self):
-        coord_a = DimCoord(np.arange(5), units="Pa")
```

Comment:
Note that, the units were originally wrong here, so I swapped them around to be as expected.